extensions/job_management/standard/recommendations/job-management/definition/REC_response-cwl.adoc

```diff
@@ -0,0 +1,9 @@
+[[rec_deploy-replace-undeploy_package_response-cwl]]
+[recommendation]
+====
+[%metadata]
+label:: /rec/deploy-replace-undeploy/package/response-cwl
+
+part:: If a process deployed as a <<rc_cwl,CWL>>, implementations SHOULD consider supporting the <<rc_cwl,CWL>> encoding.
+
```

Comment:
In development for CWL v1.3 we have introduced the `CWLInputFile` as a formal definition, https://github.com/common-workflow-language/cwl-v1.3/blob/8512c8b9ed0e8d64736ab878d783be8ebe5a4f27/Base.yml#L486C9-L486C17

Reply:
The corresponding definition should be generated for the JSON-schema representation in order to link it. Technically, that could be used as `Content-Schema` definition, but most implementations will expect the contents to be a JSON-schema rather than a schema-salad definition.